### DISCUSSION TITLE
Add folder / folder structure

### DISCUSSION THREAD

**Discussion by kbkozlev** [outline/outline#7494] [Ideas]
It would be a good Idea to be able to add folders in order to group different files per topic.
The folders should have a name, a link and perhaps some kind of colour or emoji so that they can be differentiated from other files.

Example:
Inside the collection “Home server”
I have a main file explaining what it is and all the services configured to run on it.
Below the main file but inside the collection, I have a folder “Docker” that contains all files related to services running inside of docker.
In the main file, I can reference the folder with the link.

Essentially, the folder, is a subgroup of the collection.
We can already assign files to other files (nesting them), but If I would like to organize my file structure like that, I would just have an empty top file with a name and multiple referenced files beneath it. 
Instead of an empty file, I can have a folder with a name that does the same thing, but is much cleaner.

**Comment by Rodaviva29** [comment]
Yes, please!
